agent/dockerclient/versions.go

```diff
@@ -32,15 +41,42 @@ const (
 	Version_1_30 DockerVersion = "1.30"
 	Version_1_31 DockerVersion = "1.31"
 	Version_1_32 DockerVersion = "1.32"
+	Version_1_33 DockerVersion = "1.33"
+	Version_1_34 DockerVersion = "1.34"
+	Version_1_35 DockerVersion = "1.35"
+	Version_1_36 DockerVersion = "1.36"
+	// MinDockerAPIVersion is the min Docker API version supported by agent
+	MinDockerAPIVersion = Version_1_24
```

Comment:
Question (non-blocking): if the minimum docker API version is 1.24, do we still need all of the docker versions < 1.24?